spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1,8 +1,8 @@
 require "spec"
 require "../../../src/compiler/crystal/formatter"
 
-private def assert_format(input, output = input, strict = false, file = __FILE__, line = __LINE__)
-  it "formats #{input.inspect}", file, line do
+private def assert_format(input, output = input, strict = false, file = __FILE__, line = __LINE__, focus : Bool = false)
+  it "formats #{input.inspect}", file, line, focus: focus do
```

Comment:
I'm not sure about forwarding the `focus` parameter. Should we do that for all helper methods that define an `it` block?
I think it's just as easy to wrap the code into "describe focus: true do end" if you want to focus a specific example (it's even very easy to expand to multiple ones by moving the end).